content/doc/tutorials/build-a-node-js-and-react-app-with-npm.adoc

```diff
@@ -86,48 +71,40 @@ GitHub account and then cloning this fork locally.
+
+1. Obtain the latest Jenkins instance, customized for this tutorial, by cloning the link:https://github.com/jenkins-docs/quickstart-tutorials.git[quickstart-tutorials repository].
+2. After cloning, navigate to the `quickstart-tutorials` directory, and execute the command `docker compose up -d node` to run the example.
+3. Once the containers are running successfully (you can verify this with `docker compose ps`), the controller can be accessed at http://localhost:8080.
+
+If you are unable to install `docker compose` on your machine for any reason, you can still run the example in the cloud for free thanks to link:https://www.gitpod.io/[GitPod]. GitPod link:https://www.gitpod.io/pricing[is free] for 50 hours per month.
+You need to link it to your GitHub account so you can run the example in the cloud.
+Click on link:https://gitpod.io/?autostart=true#https://github.com/jenkins-docs/quickstart-tutorials[that link] to open a new browser tab with a GitPod workspace where you'll be able to start the Jenkins instance, and run the rest of the tutorial.
+
+Now, log in using the `admin` username and `admin` password.
 
 === Create your Pipeline project in Jenkins
 
-. Go back to Jenkins, log in again if necessary and click *create new jobs*
-  under *Welcome to Jenkins!* +
-  *Note:* If you don't see this, click *New Item* at the top left.
-. In the *Enter an item name* field, specify the name for your new Pipeline
-  project (e.g. `simple-node-js-react-npm-app`).
-. Scroll down and click *Pipeline*, then click *OK* at the end of the page.
-. ( _Optional_ ) On the next page, specify a brief description for your Pipeline
-  in the *Description* field (e.g. `An entry-level Pipeline demonstrating how to
-  use Jenkins to build a simple Node.js and React application with npm.`)
-. Click the *Pipeline* tab at the top of the page to scroll down to the
-  *Pipeline* section.
-. From the *Definition* field, choose the *Pipeline script from SCM* option.
-  This option instructs Jenkins to obtain your Pipeline from Source Control
-  Management (SCM), which will be your locally cloned Git repository.
-. From the *SCM* field, choose *Git*.
-. In the *Repository URL* field, specify the directory path of your locally
-  cloned repository <<fork-sample-repository,above>>,
-  which is from your user account/home directory on your host machine, mapped to
-  the `/home` directory of the Jenkins container - i.e.
-* For macOS - `/home/Documents/GitHub/simple-node-js-react-npm-app`
-* For Linux - `/home/GitHub/simple-node-js-react-npm-app`
-* For Windows - `/home/Documents/GitHub/simple-node-js-react-npm-app`
-. Click *Save* to save your new Pipeline project. You're now ready to begin
-  creating your `Jenkinsfile`, which you'll be checking into your locally cloned
-  Git repository.
+. In Jenkins, select *New Item* under *Dashboard >* at the top left.
+. Enter your new Pipeline project name in *Enter an item name* (e.g. `simple-node-js-react-npm-app`).
```

Comment:
```suggestion
. Enter your new Pipeline project name, such as `simple-node-js-react-npm-app`, in *Enter an item name*.
```
Suggestion to rearrange the wording and [remove `e.g.`](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg)